CONTRIBUTING.md

```diff
@@ -142,13 +142,17 @@ This is usually the file you want to create to test certain compile behavior wit
 - Verify the output, check in the `jscomp/test/my_file_test.ml` and `jscomp/test/my_file_test.js` to version control. The checked in `.js` file is essential for verifying regressions later on.
 - Eventually check in other relevant files changed during the rebuild (depends on your compiler changes).
 
-## Contribute to the BS Playground Bundle
+## Contribute to the ReScript Playground Bundle
 
 > Note: These instructions are designed for building the 4.06 based version of ReScript (ReScript v6).
 
 The "Playground bundle" is the BS compiler compiled to JavaScript, including all necessary dependency files (stdlib / belt etc). It is useful for building tools where you want to compile and execute arbitrary Reason / OCaml in the browser.
 
-The ReScript source code is compiled with a tool called [JSOO (js_of_ocaml)](https://ocsigen.org/js_of_ocaml/3.5.1/manual/overview), which uses OCaml bytecode to compile to JavaScript and is part of the bigger OCaml ecosystem. Before we can compile anything, we need to install the required tools (requires [`opam`](https://opam.ocaml.org/doc/Install.html) to be installed):
+The ReScript source code is compiled with a tool called [JSOO (js_of_ocaml)](https://ocsigen.org/js_of_ocaml/3.6.0/manual/overview), which uses OCaml bytecode to compile to JavaScript and is part of the bigger OCaml ecosystem.
+
+We actually ship a bytecode version of `js_of_ocaml`, so you actually don't need to install any third party dependencies. This is usually for basic usage if you don't care about the compilation speed or a fast feedback loop (e.g. CI).
+
+For faster compilation times, we recommend to install a proper `jsoo` executable:
 
```

Comment:
this is a one time effort, I suggest we just use the vendored bytecode to avoid opam toolchain.
IIUC, if we don't need jsoo (use the vendored bytecode instead), we don't need document opam either, is it correct?

Reply:
it's mostly for those ppl who need to work on the bundle (like me). The development feedback loop (when working locally) is around 3min slower when compiling with the bytecode version.

Using the natively built version of jsoo is around 30 seconds, which is already really slow, but way better than 3 minutes.

The bytecode build is fine for processes like CI (and also way better due to zero-deps).